docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md

```diff
@@ -0,0 +1,254 @@
+
+[HAProxy](https://docs.haproxy.org/2.6/intro.html) is open source software that provides a high availability load balancer and proxy server for TCP and HTTP-based applications that spreads requests across multiple servers.
+
+The Sumo Logic App for HAProxy helps you monitor activity in HAProxy. The preconfigured dashboards provide information about site visitors, including location of visitors, HTTP Error codes percentage, Backend and Frontend server statistics.
+
+HAProxy logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/HAProxy-OpenTelemetry/HAProxy-Schematics.png' alt="Schematics" />
+
+## HAProxy Log Types
+
+The app supports Logs from the open source version of HAProxy. The App is tested on the 2.3.9 version of HAProxy.
+
+The HAProxy logs are generated in files as configured in the configuration file `/etc/haproxy/haproxy.cfg` ([learn more](https://www.haproxy.com/blog/introduction-to-haproxy-logging/)).
+
+## Fields Create in Sumo Logic for HAProxy
+
+Following are the [Fields](/docs/manage/fields/) which will be created as part of HAProxy App install if not already present.
+
+- `webengine.cluster.name`. User configured. Enter a name to identify the Haproxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- `webengine.system`. Has fixed value of **haproxy**
+- `sumo.datasource`. Has fixed value of **haproxy**
+
+## Prerequisites
+
+This section provides instructions for configuring log collection for HAProxy running on a non-Kubernetes environment for the Sumo Logic App for HAProxy.
+
+By default, HAProxy logs are forwarded to Syslog. This needs to be changed to send the logs to files. Configuration in the file `/etc/haproxy/haproxy.cfg` is needed to be modified to send logs to files.
+
+HAProxy logs have several levels of verbosity. To select a level, set your loglevel to one of the following:
+
+- emerg - Errors such as running out of operating system file descriptors.
+- alert - Some rare cases where something unexpected has happened, such as being unable to cache a response.
+- info - TCP connection and http request details and errors.
+- err - Errors such as being unable to parse a map file, being unable to parse the HAProxy configuration file, and when an operation on a stick table fails.
+- warning - Certain important, but non-critical, errors such as failing to set a request header or failing to connect to a DNS nameserver.
+- notice - Changes to a server's state, such as being UP or DOWN or when a server is disabled. Other events at startup, such as starting proxies and loading modules are also included. Health check logging, if enabled, also uses this level.
+- debug - Complete information, useful for development/testing.
```

Comment:
@himanshu219 @sumoanema @SumoLogic/sumoappdev This list doesn't match the one on [HAProxy's site](https://www.haproxy.com/blog/introduction-to-haproxy-logging/). Can you advise?